ios/brave-ios/Sources/BraveShared/BraveURLs.swift

```diff
@@ -40,6 +40,9 @@ extension URL {
     public static let braveVPNLinkReceiptDev = URL(
       string: "https://account.brave.software/?intent=connect-receipt&product=vpn"
     )!
+    public static let braveVPNRefreshCredentials = URL(
+      string: "https://account.brave.com/?intent=recover&product=vpn&ux=mobile"
```

Comment:
`https://account.brave.com/?intent=recover&product=leo&ux=mobile` for Leo credentials refresh?

Reply:
Good catch thanks. Made a mistake there. Change at https://github.com/brave/brave-core/pull/23649/commits/c0ea6fd20da1c4c02f64cbb7f0d918b0f032250b